noxfile.py

```diff
@@ -593,6 +598,77 @@ def test_version_limits(session: nox.Session):
         _run_cargo(session, "check", env=env, expect_error=True)
 
 
+@nox.session(name="test-feature-powerset", venv_backend="none")
+def test_feature_powerset(session: nox.Session):
+    assert toml is not None, "requires Python 3.11 or toml package"
+
+    with (PYO3_DIR / "Cargo.toml").open("rb") as cargo_toml_file:
+        cargo_toml = toml.load(cargo_toml_file)
+
+    EXCLUDED_FROM_FULL = {
+        "nightly",
+        "extension-module",
+        "full",
+        "default",
+        "auto-initialize",
+        "generate-import-lib",
+        "multiple-pymethods",  # TODO add this after MSRV 1.62
+    }
+
+    features = cargo_toml["features"]
+
+    full_feature = set(features["full"])
+    expected_full_feature = {
```

Comment:
Could this be rewritten as a set expression? Something like

```python
expected_full_feature = features.keys() - abi3_version_features - EXCLUDED_FROM_FULL
```

Reply:
Done 👍